### PR TITLE
 Changing the IP range of pods from 100.64.48.0/20 to 100.65.16.0/20 Fixes #1461

### LINKED ISSUE #1461: FAST: IPAM conflict between GKE and dataplatform dev subnets
`fast/stages/2-networking-a-peering/data/subnets/dev/dev-gke-nodes-ew1.yaml1`:
```
secondary_ip_ranges:
   pods: 100.64.0.0/16
   services: 192.168.1.0/24
```
`fast/stages/2-networking-a-peering/data/subnets/dev/dev-dataplatform-ew1.yaml`:
```
secondary_ip_ranges:
   pods: 100.64.48.0/20
   services: 100.64.64.0/24
```
overlap on the `100.64.0.0/16` range.

### PR DESCRIPTION
**Pull request by arvindag07**
-> Changing the IP range of pods from 100.64.48.0/20 to 100.65.16.0/20 in `fast/stages/2-networking-a-peering/data/subnets/dev/dev-dataplatform-ew1.yaml` as there is an overlap in 100.64.0.0/16 range with `fast/stages/2-networking-a-peering/data/subnets/dev/dev-gke-nodes-ew1.yaml1` "Fixes [#1461](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/issues/1461)"

### CODE REVIEW

File: fast/stages/2-networking-a-peering/data/subnets/dev/dev-dataplatform-ew1.yaml [hunk 1]
```diff
@@ -4,5 +4,5 @@ region: europe-west1
 description: Default subnet for dev Data Platform
 ip_cidr_range: 10.128.48.0/24
 secondary_ip_ranges:
-   pods: 100.64.48.0/20
+   pods: 100.65.16.0/20
    services: 100.64.64.0/24
```
Comment (juliocc) [inline fast/stages/2-networking-a-peering/data/subnets/dev/dev-dataplatform-ew1.yaml:8]: can you change services to 100.65 too?

**Review by juliocc — APPROVED** [review]

**Review by juliocc — CHANGES_REQUESTED** [review]
and you need to do the same in the other networking stages

File: fast/stages/2-networking-a-peering/data/subnets/dev/dev-dataplatform-ew1.yaml (hunk 1 above)
Comment (arvindag07) [reply fast/stages/2-networking-a-peering/data/subnets/dev/dev-dataplatform-ew1.yaml:8]: Ack Julio. 
Changing the service range in `fast/stages/2-networking-a-peering/data/subnets/dev/dev-gke-nodes-ew1.yaml1` to 100.65.64.0/24
Service range in `fast/stages/2-networking-a-peering/data/subnets/dev/dev-dataplatform-ew1.yaml` is 100.64.64.0/24 which is unchanged as these ranges do not overlap.

**Comment by arvindag07** [comment]
Changed the IP ranges in all branches

**Comment by drebes** [comment]
Can you edit the original description and add "Fixes #1461"? Thank you.

**Comment by juliocc** [comment]
Sorry, I didn't explain myself clearly.

Let's make sure DP and GKE use the same ranges in all networking stages. 

For DP we want (@lcaggio can you LGTM these ranges?):
- Primary range: 100.127.48.0/24 (unchanged)
- Pods secondary range: 100.64.0.0/24 (was 100.64.48.0/20)
-  Services secondary range: 100.64.1.0/24 (was 100.64.64.0/24)

For GKE we want (@danielmarzini can you LGTM these ranges?):
- Primary range: 100.127.49.0/24 (was 100.64.0.0/24)
- Pods secondary range: 100.65.0.0/24 (was 100.64.0.0/16)
- Services secondary range: 100.65.1.0/24 (wass 192.168.1.0/16)

This means updating the following subnets with the ranges above:
- 2-networking-a-peering dev-dataplatform-ew1.yaml
- 2-networking-a-peering dev-gke-nodes-ew1.yaml
- 2-networking-b-vpn dev-dataplatform-ew1.yaml
- 2-networking-b-vpn dev-gke-nodes-ew1.yaml
- 2-networking-c-nva dev-dataplatform-ew1.yaml
- 2-networking-d-separate-envs dev-dataplatform-ew1.yaml
- 2-networking-e-nva-btp dev-dataplatform-ew1.yaml

Stages c, d, and e don't have the GKE subnet. We'll track that in #1481 

@simonebruzzechesse @sruffilli can you quickly make sure these ranges are ok in your net stages?

**Comment by arvindag07** [comment]
Thanks for the detailed comment. Made all the suggested changes as mentioned in https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/pull/1477#issuecomment-1612846907. Please review

**Review by juliocc — APPROVED** [review]

**Pull request merged**